### DISCUSSION TITLE
Remove fallbacks for flex and grid?

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#553] [Content]
Should we remove these two pages?

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Backwards_Compatibility_of_Flexbox
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid_and_Progressive_Enhancement

Both have been fully supported for 5+ years in evergreen browsers.

**Comment by teoli2003** [comment]
I would be curious to see their popularity before taking a final decision. @Rumyra ?

If we keep them, we should:
- remove them from the sidebar
- add a comment at the top saying you don't need these tips if you are developing for the web.

If we don't keep them, we should redirect them to the respected overview page. (And update the sidebar, too)

My heart is saying we should _remove them and redirect_.
